index.html

```diff
@@ -3,44 +3,148 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>Felony Disenfranchisement</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Fjalla+One|Exo:400,600|Ropa+Sans' rel='stylesheet' type='text/css'>
   <style>
     .header {
-      text-align: center;
+      text-align: left;
     }
+    table {
+    border-collapse: collapse;
+    width: 100px;
+    }
+
+    th, td {
+    padding: 8px;
+    text-align: left;
+    border-bottom: 1px solid #ddd;
+    }
+
+    tr:hover td{font-weight: bold;
+    }
+
+    body {
+      background-color: #4C4C4C;
+          }
+
+    th {
+       font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+        width: 400px;
+                } 
+   h1{
+       font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+                }   
+    p,td {
+       font-family: 'Exo', sans-serif;
+        color: #B7B7B7
```

Comment:
Your aesthetic here is great. One thing to watch out for, though is legibility. I'd classify Exo as a "display" font, best suited to headings and the like. It's great, but best when paired with a more easily read font for the body content. Additionally, the color is very low-contrast as compared to the background. This is a particular issue for older users, or those with a visual impairment. WebAIM.org provides a color contrast checker that can test your color choices against the WCAG 2.0 web accessibility standard (ideally your choice would meet the AA level for normal text). Find that here: http://webaim.org/resources/contrastchecker/